ngx_pq_module.c

```diff
@@ -1237,6 +1239,15 @@ static char *ngx_pq_argument_output_loc_conf(ngx_conf_t *cf, ngx_pq_query_t *que
             query->string = e[j].value;
             continue;
         }
+        if (str[i].len > sizeof("notfound=") - 1 && !ngx_strncasecmp(str[i].data, (u_char *)"notfound=", sizeof("notfound=") - 1)) {
```

Comment:
It would be better to use the name `no_data_found`, as when catching the corresponding exception in `plpgsql`.